radio/src/gui/colorlcd/preview_window.h

```diff
@@ -181,21 +181,18 @@ class ThemedTextEdit : public TextEdit
   public:
     ThemedTextEdit(Window *parent, const rect_t &rect, char *text, 
                    int colorBackgroundIndex, int colorTextIndex) :
-      TextEdit(parent, rect, text, strlen(text)),
+      TextEdit(parent, rect, text, strlen(text), NO_FOCUS),
       _colorBackgroundIndex(colorBackgroundIndex),
       _colorTextIndex(colorTextIndex)
     {
-      setBackgroundHandler([=] (FormField *field) {
-        return COLOR(_colorBackgroundIndex);
-      });
     }
 
     void paint(BitmapBuffer *dc) override
     {
       colorMaintainer.applyColorValues();
+      lv_obj_set_style_bg_color(lvobj, makeLvColor(COLOR(_colorBackgroundIndex)), LV_PART_MAIN);
+      lv_obj_set_style_text_color(lvobj, lv_color_white(), LV_PART_MAIN);
       FormField::paint(dc);
-
-      dc->drawSizedText(FIELD_PADDING_LEFT, FIELD_PADDING_TOP, value, length, COLOR(_colorTextIndex));
```

Comment:
The text is not displayede anymore?

Reply:
Value is displayed by the TextEdit control - this line was trying to overwrite the value in a different color.
The 'Edit' and 'Focus' boxes did not reflect how they actually look - now they should.

Reply:
ok, strange